pkg/skuba/actions/cluster/init/manifests.go

```diff
@@ -18,14 +18,18 @@
 package cluster
 
 const (
+	CriConfFolderReadme = `This directory contains CRI-O configuration files that are uploaded to CaaSP nodes. All the files (except this README) will be uploaded to the /etc/crio/conf.d/ folder. If it is required user custom files name as 99-custom.conf
+	`
 	criDockerDefaultsConf = `## Path           : System/Management
 ## Description    : Extra cli switches for crio daemon
 ## Type           : string
 ## Default        : ""
 ## ServiceRestart : crio
 #
 CRIO_OPTIONS=--pause-image={{.PauseImage}}{{if not .StrictCapDefaults}} --default-capabilities="CHOWN,DAC_OVERRIDE,FSETID,FOWNER,NET_RAW,SETGID,SETUID,SETPCAP,NET_BIND_SERVICE,SYS_CHROOT,KILL,MKNOD,AUDIT_WRITE,SETFCAP"{{end}}`
-	criDefaultsConf = `{{if not .StrictCapDefaults}}[crio.runtime]
+	criDefaultsConf = `# PLEASE DON'T EDIT THIS FILE! 
```

Comment:
nit: there is a trailing whitespace after the exclamation point.

Reply:
Fixed